spec/unit/transaction/report_spec.rb

```diff
@@ -488,6 +463,7 @@ def add_statuses(count, type = :file)
       trans = catalog.apply
 
       @report = trans.report
+      @report.add_times(:total, "8675") #Report total is now measured, not calculated.
```

Comment:
Why do we need to add total here if we're not checking it elsewhere in this test? Was this just leftover from earlier testing?

Reply:
It is actually needed in the [later test](https://github.com/kris-bosland/puppet/blob/5c092467789093d87228c1f471a469d314e3ae4f/spec/unit/transaction/report_spec.rb#L513), we have to add a value directly because the total is no longer adding the values that are already added in the test.